packages/docs/stories/Welcome.mdx

```diff
@@ -0,0 +1,83 @@
+</Chapter><br/>
+
+<Chapter title="Offerings">
+- Components, Design-Tokens, Templates and Assets
+- Modular and flexible UI components which can be put together like Lego blocks.
+  - The components exist twice; as web- (development) and as Figma-components (design) which are always in-sync and have the exact same anatomy.
+  - [Angular](https://angular.io), [React](https://react.dev) and [Vue](https://vuejs.org) are first-class citizens. Due to the nature of web-components which are framework agnostic they run in all frameworks or without any. But we decided to give special support to those popular frameworks so they profit from better DX.
+  - The web-components are accessible (A11Y)
+- Auto-completion in TS, JS and HTML (VSCode)
+- Customizability: easy and profound options for styling and branding via DesignTokens, CSS Variables and CSS Parts.
+- Light and dark themes are included
+- Material icons
+- Support for all major browsers; tested with every release.
+- Basic CSS styles for an easy application start
+- Templates for typical and recurring UIs which can be copied.
+</Chapter><br/>
+
+<Chapter title="Foundations">
+- The [web-components](https://developer.mozilla.org/en-US/docs/Web/API/Web_components) are build upon popular technologies, especially [Lit](https://lit.dev) and [Shoelace](https://shoelace.style) 🙏
+- [Material Icons](https://fonts.google.com/icons) from Google 🙏
+</Chapter><br/>
+
+<Chapter title="Trust">
+_Using a base library for an application is and should be a careful and very well-considered process. Besides the technology stack and features another key factor is trust.
+Why do we think you should trust Synergy?_
+- We also considered our base technologies very carefully and decided for the ones which are most popular and easy to maintain if necessary
+- We are building on web standards
+- Every used technology is independent from other technologies as much as possible and meaningful
+- In development, dependencies are updated on a regular basis
+- It is our aim to provide a long lasting library with a stable API.
+- Synergy has a dedicated team with almost a single responsibility into this project
+- All team members are very experienced designers and web developers with a long history in developing design systems
+- Synergy is backed by big and strategically important products at SICK
+- Synergy is seen by top management as a very important corner stone for a successful and versatile product palette
+- We are putting a lot of effort into quality assurance
+- We are listening very carefully to our stakeholders needs
+- We made everything transparent: code (open source), roadmap, issues, aims, guidelines, decision, ...
+</Chapter><br/>
+
+<Chapter title="Roadmap">
+What you're seeing here and now is just the status quo of provided deliveries. New features and components are developed continuously.
```

Comment:
I would write: 
Our product roadmap is an up-to-date report on the work we’re doing and planning to do. New features and components are added continuously.